apps/website-25/src/pages/contact.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Breadcrumbs,
+  HeroH1,
+  HeroSection,
+  Section,
+} from '@bluedot/ui';
+import Head from 'next/head';
+import { ROUTES } from '../lib/routes';
+
+const CURRENT_ROUTE = ROUTES.contact;
+
+const ContactPage = () => {
+  return (
+    <div>
+      <Head>
+        <title>Contact us | BlueDot Impact</title>
+        <meta name="description" content="Contact BlueDot Impact, the team behind AI Safety Fundamentals." />
+      </Head>
+      <HeroSection>
+        <HeroH1>Contact us</HeroH1>
+      </HeroSection>
+      <Breadcrumbs route={CURRENT_ROUTE} />
+      <Section className="intro-section" title="How to get in touch">
+        <div className="intro-section__container grid lg:grid-cols-2 gap-8">
+          <img className="intro-section__image block lg:hidden w-full rounded-2xl" src="/images/team/team_2.jpg" alt="BlueDot Impact team" />
+          <div className="intro-section__text-container flex flex-col gap-space-between">
+            <p>We’re BlueDot Impact, the team behind AI Safety Fundamentals. You can find more about us on our <a href={ROUTES.home.url}>homepage</a>.</p>
+            <p>We love hearing from people, and are keen for people to reach out to us with any questions or feedback!</p>
+            <p>Email us at <a href="mailto:team@bluedot.org">team@bluedot.org</a>.</p>
+          </div>
+          <img className="intro-section__image hidden lg:block w-full rounded-2xl" src="/images/team/team_2.jpg" alt="BlueDot Impact team" />
```

Comment:
if we use `flex` instead of `grid`, we can just adapt [flex order](https://tailwindcss.com/docs/order) at different breakpoints, so we don't need to code dupe
```suggestion
          <img className="intro-section__image order-first lg:order-last w-full rounded-2xl" src="/images/team/team_2.jpg" alt="BlueDot Impact team" />
```

Reply:
Oh neat, I did not know about `order`! Yeah this is way better, will fix.